### ISSUE TITLE
Create life linked minigame.

### ISSUE THREAD

**Issue by homiemace** [monke-mob/monke-activities#23]
Assemble in a team of two players who share a life link. Your mission is to eliminate all the other teams of two while ensuring that none of you dies in the process. In case of an odd number of players, one player could be left alone, or a group of three to four players may form a team. You earn currency by killing other players, but winning the minigame by eliminating all opponents grants you more currency than just killing others.

**Comment by Lanred-Dev** [comment]
I don't think it would be a good idea to leave a player alone as that would then ruin the challenge for that player.

**Comment by Lanred-Dev** [comment]
The only thing that this mode lacks currently is the combat. What is the combat going to be like?
